### PR TITLE
Feature/setup testing

### PR DESCRIPTION
**Pull request by andreamartz**
Addresses Issue #255:
1. Installed jest and jest-watch-typeahead plugin
2.  Added sample test that passes
3. Added a jest config file that ignores node_modules and cypress tests 
4. Changed the scripts in package.json (these are just ideas I'm proposing)
- "test" was previously a script for running end-to-end tests with Cypress
- I propose renaming that script something like "e2e:test" and reserving "test" for tests run with jest
- Also, I added a "test:watch" script for running jest tests in watch mode.

### CODE REVIEW

**Comment by drewclem** [comment]
Thank you for this @andreamartz !!

What steps would we need to take to create a folder for our tests to live in? I can see the number of tests growing and it'd be easier to manage if they had their own directory.

Just ran my first test and it passed!

File: package.json [hunk 1]
```diff
@@ -44,7 +44,9 @@
     "zustand": "^3.3.1"
   },
   "devDependencies": {
+    "dotenv": "^8.2.0",
```
Comment (nickytonline) [inline package.json:47]: Just merged some changes into your branch I made yesterday @andreamartz.

**Comment by andreamartz** [comment]
@drewclem,

Glad to hear the first test passed!

Two things:
1. Could you tell me what is failing with Vercel? When I click on the `Details` link, I'm taken to the login page for Vercel. I haven't created an account and tried to log in, but if you think I'd have access to the information I'm looking for, I can do that.

2. As for your question about setting up tests, having a single testing directory is one option, as you mentioned. You could do that certainly. Here is another approach to consider:

* From what I understand, colocating the tests to be near the components and utility functions they are associated with has some advantages as a project grows.
*  Nick said colocating tests is how they do it at Forem. See an example from Forem [here](https://github.com/forem/forem/tree/master/app/javascript/CommentSubscription).
* Nick also pointed me to this [article](https://kentcdodds.com/blog/colocation#tests) by Kent C. Dodds about colocation. There is a section in the middle of it about colocation of tests.

**Comment by drewclem** [comment]
@andreamartz That make sense to me!

**Pull request merged**